crates/gpui/src/platform/windows/platform.rs

```diff
@@ -47,6 +47,7 @@ pub(crate) struct WindowsPlatform {
     foreground_executor: ForegroundExecutor,
     text_system: Arc<dyn PlatformTextSystem>,
     dispatch_event: OwnedHandle,
+    clipboard: RefCell<ClipboardContext>,
```

Comment:
This change is unnecessary. The clipboard context is a noop: https://github.com/alacritty/copypasta/blob/master/src/windows_clipboard.rs#L22-L24